web-app/js/portal/ui/openlayers/layer/NcWMS.js

```diff
@@ -102,9 +102,7 @@ OpenLayers.Layer.NcWMS = OpenLayers.Class(OpenLayers.Layer.WMS, {
         });
     },
 
-    _timeSeriesDatesLoaded: function(datesWithData) {
```

Comment:
> I actually think the function names make perfect sense...

Well, that may be the case for someone familiar with the code, but just saying it took me about 30 minutes of reviewing this class to figure out what was going on here, so that suggests that some refactoring might be in order (more moving things in to a dedicated class rather than just renaming, I think).

As always, I'm ok to leave it as is, since it's a matter of opinion.